ansible/minigraph/switch4.xml

```diff
@@ -0,0 +1,916 @@
+<DeviceMiniGraph xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="Microsoft.Search.Autopilot.Evolution">
```

Comment:
Could you summarize what are special in this minigraph comared with others? If no, you may reuse existing one.